two_factor/templates/two_factor/admin/_base.html

```diff
@@ -1,23 +1,2 @@
+{% extends "admin/login.html" %}
```

Comment:
This template is used by all 2fa admin pages, not only the login one. It should not inherit from the login page template

Reply:
```suggestion
{% extends "admin/base_site.html" %}
```

Possibly now `two_factor/templates/two_factor/admin/login.html` now needs to extend from `admin/login.html` rather than `two_factor/admin/_base.html`